ntbs-service/Pages/ValidationModel.cs

```diff
@@ -64,5 +69,60 @@ public void SetAndValidateDate(object model, string key, FormattedDate formatted
                 return;
             }
         }
+
+        protected ContentResult ValidateYearComparison(string yearToValidate, int yearToCompare)
+        {
+            if (IsValidYear(yearToValidate)) 
+            {
+                if (int.Parse(yearToValidate) < yearToCompare)
+                {
+                    return Content(ValidationMessages.ValidYearLaterThanBirthYear(yearToCompare));
+                }
+                else
+                {
+                    return ValidContent();
+                }
+            } 
+            else
+            {
+                return Content(ValidationMessages.ValidYear);
+            }
+        }
+
+        private bool IsValidYear(string year)
+        {
+            if (int.TryParse(year, out int parsedYear))
+            {
+                return parsedYear >= ValidDates.EarliestYear && parsedYear <= DateTime.Now.Year;
+            }
+            else 
+            {
+                return false;
+            }
+        }
+
+        protected void ValidateYearComparisonOnModel(object model, string key, string yearToValidate, int? yearToCompare)

```

Comment:
I decided not do any commonising, but instead just add "onModel" to the name to distinguish the dynamic ones. We can always refactor later, but there are subtle enough differences that it's probably worth treating them separately.